k8s/namespaces/wa/wa-message-find-problem-messages/aat-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-message-find-problem-messages
+spec:
+  values:
+    job:
+      schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
```

Comment:
Could make it less frequent to start like every 15 mins?  I.e. to see if it is working ok.

Reply:
Done